website/blog/2024-04-22-extended-attributes.md

```diff
@@ -84,7 +80,7 @@ All you need to do is configure an environment as staging and enable the **Defer
 
 ## Upgrading on a curve
 
-Lastly, let’s consider a more specialized use case. Imagine we have a "tiger team" (consisting of a lone analytics engineer named Dave) tasked with upgrading from dbt version 1.6 to the new **Latest release track**, to take advantage of new features and performance improvements. We want to keep the rest of the data team being productive in dbt 1.6 for the time being, while enabling Dave to upgrade and do his work with Latest (and greatest) dbt.
+Lastly, let’s consider a more specialized use case. Imagine we have a "tiger team" (consisting of a lone analytics engineer named Dave) tasked with upgrading from dbt version 1.6 to the new **[Latest release track](/docs/dbt-versions/cloud-release-tracks)**, to take advantage of new features and performance improvements. We want to keep the rest of the data team being productive in dbt 1.6 for the time being, while enabling Dave to upgrade and do his work with Latest (and greatest) dbt.
```

Comment:
This article was written during the Keep on Latest Version phase, so it never talked about versionless and explaining that we used to call it versionless was going to mess with the screenshots! So I've just linked to release tracks.